src/Components/Components/src/Reflection/PropertyGetter.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.AspNetCore.Components.Reflection;
+
+internal sealed class PropertyGetter
```

Comment:
This is a `copy and modification` of `PropertySetter`

Reply:
https://github.com/dotnet/aspnetcore/blob/main/src/Components/Components/src/Reflection/PropertySetter.cs